Lib/test/test_pathlib.py

```diff
@@ -1643,6 +1643,24 @@ def test_rmdir(self):
         self.assertFileNotFound(p.stat)
         self.assertFileNotFound(p.unlink)
 
+    def test_link(self):
+        P = self.cls(BASE)
+        p = P / 'fileA'
+        size = p.stat().st_size
+        # linking to another path.
+        q = P / 'dirA' / 'fileAA'
+        try:
+            p.link(q)
+        except PermissionError as e:
+            self.skipTest('os.link(): %s' % e)
+        self.assertEqual(q.stat().st_size, size)
```

Comment:
To test that the target is really a link, you should probably use [os.path.samefile](https://docs.python.org/3/library/os.path.html#os.path.samefile).